Containers/talk/Dockerfile

```diff
@@ -1,44 +1,45 @@
 FROM nats:2.9.19-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
+
+FROM alpine:3.18.2 as janus
+ARG JANUS_VERSION=v0.13.4
+WORKDIR /src
+RUN apk add --no-cache \
+        ca-certificates \
+        git autoconf automake build-base pkgconfig libtool util-linux \
+        glib-dev zlib-dev openssl-dev jansson-dev libnice-dev libconfig-dev libsrtp-dev gengetopt-dev \
+        libwebsockets-dev; \
+        git clone --recursive https://github.com/meetecho/janus-gateway -b $JANUS_VERSION /src; \
+        /src/autogen.sh; \
+        /src/configure; \
+        make; \
+        make install; \
+        make configs; \
+        rename -v ".jcfg.sample" ".jcfg" /usr/local/etc/janus/*.jcfg.sample
+
 FROM coturn/coturn:4.6.2-alpine
 USER root
 # Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
-
-COPY --from=nats /nats-server /usr/local/bin/nats-server
-COPY --from=signaling /usr/bin/nextcloud-spreed-signaling /usr/local/bin/nextcloud-spreed-signaling
-
-COPY --chmod=775 start.sh /start.sh
-COPY --chmod=664 supervisord.conf /supervisord.conf
+ARG ALPINE_VERSION=3.18
 
 RUN set -ex; \
     grep VERSION_ID /etc/os-release | grep -q "$ALPINE_VERSION.[0-9]\+$"; \
     apk add --no-cache \
         ca-certificates \
         tzdata \
         bash \
-        janus-gateway \
         openssl \
         supervisor \
         bind-tools \
         netcat-openbsd \
-        shadow \
-        util-linux \
-        build-base \
-        wget \
-        lua5.3-dev \
-        luarocks5.3; \
+        \
+        glib zlib libssl3 libcrypto3 jansson libnice libconfig libsrtp \
+        libwebsockets \
```

Comment:
we maybe need to change to optional modules

Reply:
the structureag image also contains: `libcurl libogg`

Reply:
ogg adds `Voice Mail` and curl `Sample event handler`/`Sample event handler`

Reply:
If I read https://github.com/strukturag/nextcloud-spreed-signaling#setup-of-janus, we don't need those three modules

Reply:
all right, then lets remove them.

Reply:
ah I see, sorry